test/pseudo-tty/no_dropped_stdio.js

```diff
@@ -12,4 +12,13 @@ let out = ('o'.repeat(48) + '\n').repeat(20);
 out += 'o'.repeat(24) + 'O';
 
 process.stdout.write(out);
-process.exit(0);
+
+// In AIX, the child exits even before the python parent
+// can setup the readloop. Provide a reasonable delay.
+if (common.isAix) {
+  setTimeout(function() {
+    process.exit(0);
+  }, 200);
```

Comment:
This looks like it would completely break the purpose of the test… if the problem really is the child exiting too quickly, maybe just set up a `setTimeout` around the whole code here for AIX.

Also, the comment sounds like this is a bug in Python on AIX, not in our tests/Node?